### ISSUE TITLE
[BOUNTY] Algorand - Best UX Application 🔮

### ISSUE THREAD

**Issue by wisthsb1519** [ethberlinzwei/Bounties#14]
# Algorand: Best UX Application 🔮

Algorand is the best way to start building your decentralized application quickly! Algorand lets you build any type of layer 2 application on a blockchain network with near-instant transaction finality. Our JavaScript, Python, Go and Java [SDK’s](https://developer.algorand.org/docs/using-sdks-and-rest-apis) allow you to take advantage of identity, key, account and wallet management easily and effectively for your webapp.

## How to qualify

Qualifying submission should 
-    Make use of the developer tools
-    Be an innovative solution that improves the Algorand developer user experience
-    Code must be posted on Github
-    comply with the overall ETHBerlin submission guidelines

## Resources

Example solutions:
-   [Evancoin](https://medium.com/algorand/l2-applications-on-algorand-make-your-own-coin-6d62c5d5578d)
-   [ARC-01](https://medium.com/assetblock/arc-01-building-a-token-standard-proposal-on-algorand-17fe634397ce)

Community Resources:
-   [Algorand Developer Documentation](https://developer.algorand.org/)
-   [Algorand Forums](https://forum.algorand.org/)
-   [Gitter Community](https://gitter.im/algorand-ethzweihack/community)
-   [Discord](https://discord.gg/SqNRnw7)
-   [Reddit](https://www.reddit.com/r/algorand/)

## Judging Criteria

1\.     Creative: How creative or innovative is the application?
2\.     Technicals: What technical stack does the application make use of? How well is it built, technically?
3\.    Complete: How complete is this application with respect to the Algorand dev tools?

## Prizes

1\.    🥇First Place: Thule 32L Backpack + Gotenna
2\.     🥈Second Place: $200 Amazon Gift Card

**Comment by labsterx** [comment]
Is this the place to submit my project to your bounty? Here's my project on devpost: https://devpost.com/software/collectorhub

**Comment by wisthsb1519** [comment]
yes it is! awesome. come by our booth when youre ready so you can walk us through your app. we're in the courtyard
